src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -189,6 +189,39 @@
     type: command
     short-summary: List policy definitions.
 """
+helps['policy setdefinition create'] = """
+            type: command
+            short-summary: Create a policy set definition.
+            parameters:
+                - name: --definitions
+                  type: string
+                  short-summary: Policy definitions in JSON format, or a path to a file containing JSON rules.
+            examples:
+                - name: Create a policy set definition.
+                  text: |
+                    az policy setdefinition create -n readOnlyStorage --definitions \\
+                        [ \\
+                            { \\
+                                "policyDefinitionId": "/subscriptions/mySubId/providers/Microsoft.Authorization/policyDefinitions/storagePolicy" \\
+                            } \\
+                        ]
+"""
+helps['policy setdefinition delete'] = """
+    type: command
+    short-summary: Delete a policy set definition.
+"""
+helps['policy setdefinition show'] = """
+    type: command
+    short-summary: get a policy set definition.
+"""
+helps['policy setdefinition update'] = """
+    type: command
+    short-summary: Update a policy set definition.
+"""
+helps['policy setdefinition list'] = """
+    type: command
+    short-summary: List policy set definitions.
+"""
```

Comment:
There is no description for `az policy setdefinition -h`

Reply:
Done